<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>访问控制系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>181</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));
border-style:hidden</string>
    </property>
   </widget>
   <widget class="QColumnView" name="header_view">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));
border-style: hidden;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="funcstack">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <widget class="QPushButton" name="roleManage_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-style:hidden;
font: 11pt &quot;Agency FB&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));</string>
    </property>
    <property name="text">
     <string>角色管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="role_permission_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-style:hidden;
font: 11pt &quot;Agency FB&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));</string>
    </property>
    <property name="text">
     <string>权限管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userManage_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-style:hidden;
font: 11pt &quot;Agency FB&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));</string>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listManage_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-style:hidden;
font: 11pt &quot;Agency FB&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));</string>
    </property>
    <property name="text">
     <string>黑白名单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pgintegrity_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-style:hidden;
font: 11pt &quot;Agency FB&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 199, 255, 255), stop:1 rgba(0, 65, 211, 255));</string>
    </property>
    <property name="text">
     <string>数据验证</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Exit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 65, 211, 255);
color: rgb(255, 255, 255);
font: 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>中国电网</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_username">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>xxx，欢迎您</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>访问控制系统</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
